ospf6d/ospf6_interface.c

```diff
@@ -1637,12 +1637,13 @@ void ospf6_interface_start(struct ospf6_interface *oi)
 		return;
 
 	oa = ospf6_area_lookup(oi->area_id, ospf6);
-	if (oa == NULL)
+	if (oa == NULL) {
```

Comment:
This is not a correct fix. What if area already exists but the interface is not yet attached?
The correct way to fix this is to add:
```
if (oi->area)
	return;
```
after the check for `area_id_format`.